CHANGES.rst

```diff
@@ -10,7 +10,7 @@ Version 3.3
 Major Features
 ^^^^^^^^^^^^^^
 
-- Nothing so far
+- Auto-linking of patterns in minutes (e.g. issue trackers, Github repos...)
```

Comment:
I'd put it under improvements for now (and add a reference to this PR) - easier to pick stuff for this section later when preparing the release